test/e2e/network_segmentation.go

```diff
@@ -567,7 +567,7 @@ var _ = Describe("Network Segmentation", func() {
 					return connectToServer(clientPodConfig, externalIpv4, port)
 				}, 2*time.Minute, 6*time.Second).Should(Succeed())
 			},
-			Entry("by one pod with a single IPv4 address over a layer2 network",
+			XEntry("by one pod with a single IPv4 address over a layer2 network",
```

Comment:
maybe just add // TODO or FIXME and make sure we unskip this l2 egress PR @maiqueb @qinqon

Reply:
ack added a FIX ME